docs/common-ids.md

```diff
@@ -0,0 +1,20 @@
+## Common (Resource) IDs
+
+Pandora leans heavily on the concept of Resource IDs, which is a unique identifier for the Azure Resource which typically maps to the (normalized) URI.
+
+In Azure Resource Manager these tie to Resources within the Azure Resource Providers (e.g. `Microsoft.Compute`) - in Microsoft Graph these tie to the Services (such as `ServicePrincipals` or `Users`).
+
+Typically, these Resource IDs are located within a given Service (such as `Compute`), API Version (`2020-01-01`) and Resource (e.g. `DedicatedHostGroups`) - however a subset of these Resource IDs are used across many Services (such as a Network Interface, Storage Account or Virtual Machine ID).
+
+Rather than having different versions of these Resource IDs exposed per Service/API Version/Resource combination, these Common Resource IDs are instead detected by Pandora and referenced [from `hashicorp/go-azure-sdk`'s `commonids` package](XXX) instead.
```

Comment:
```suggestion
Rather than having different versions of these Resource IDs exposed per Service/API Version/Resource combination, these Common Resource IDs are instead detected by Pandora and referenced [from `hashicorp/go-azure-sdk`'s `commonids` package](https://github.com/hashicorp/go-azure-helpers/tree/main/resourcemanager/commonids) instead.
```

Reply:
🤦 good spot, ta.